cli/src/client/mod.rs

```diff
@@ -21,19 +21,21 @@ pub struct Client {
     reqwest: blocking::Client,
 }
 
+pub type GraphQLDocument = String;
```

Comment:
Can you put this in the schema module? Also add a comment explaining we need a typedef for scalars we declare in our schema.